clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -154,14 +166,88 @@ public OffsetFetchResponse(int throttleTimeMs, Errors error, Map<TopicPartition,
         this.error = error;
     }
 
+    /**
+     * Constructor without throttle time for version 8 and above.
+     * @param errors Error code on a per group level basis
+     * @param responseData Fetched offset information grouped group id
+     */
+    public OffsetFetchResponse(Map<String, Errors> errors, Map<String, Map<TopicPartition, PartitionData>> responseData) {
```

Comment:
Why do we have this constructor?

Reply:
This constructor allows us to initialize errors and the topic partitions for multiple groups. Since the KIP is for implementing batching of the `offsetFetch` API, this constructor allows for exactly that. The request allows for multiple groups to get offsets for, and the response is so we can return all of those offsets for each group requested.

Reply:
The other constructor without throttle time is for versions which didn't have throttle time. For newer constructors that will be used with newer versions, we shouldn't need that.

Reply:
Removed the constructor with `DEFAULT_THROTTLE_TIME`.